File: Mobile/src/com/esprit/app/entity/Adresse.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package com.esprit.app.entity;

public class Adresse {
    private int id;
    private String ville;
    private String rue;
    private int numMaison;
    private int idUser;

    public Adresse() {
    }

    public Adresse(String ville, String rue, int numMaison, int idUser) {
        this.ville = ville;
        this.rue = rue;
        this.numMaison = numMaison;
        this.idUser = idUser;
    }

    public Adresse(int id, String ville, String rue, int numMaison, int idUser) {
        this.id = id;
        this.ville = ville;
        this.rue = rue;
        this.numMaison = numMaison;
        this.idUser = idUser;
    }

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getVille() {
        return ville;
    }

    public void setVille(String ville) {
        this.ville = ville;
    }

    public String getRue() {
        return rue;
    }

    public void setRue(String rue) {
        this.rue = rue;
    }

    public int getNumMaison() {
        return numMaison;
    }

    public void setNumMaison(int numMaison) {
        this.numMaison = numMaison;
    }

    public int getIdUser() {
        return idUser;
    }

    public void setIdUser(int idUser) {
        this.idUser = idUser;
    }
    
    @Override
    public String toString() {
        return "Adresse{" + "ville=" + ville + ", rue=" + rue + ", numMaison=" + numMaison + ", idUser=" + idUser + '}';
    }
    
}

File: Web/src/Entity/Adresse.php
<?php

namespace App\Entity;

use Doctrine\ORM\Mapping as ORM;
use Symfony\Component\Validator\Constraints as Assert;


/**
 * @ORM\Entity
 */
class Adresse
{
    /**
     * @ORM\Id
     * @ORM\GeneratedValue
     * @ORM\Column(name="id_adresse", type="integer")
     */
    private $idAdresse;

    /**
     * @var string
     * @Assert\NotBlank(message="rue doit etre non vide")
     * @ORM\Column(name="ville", type="string", length=50, nullable=false)
     */
    private $ville;

    /**
     * @var string
     * @Assert\NotBlank(message="rue doit etre non vide")
     * @ORM\Column(name="rue", type="string", length=50, nullable=false)
     */
    private $rue;

    /**
     * @var int
     *
     * @Assert\Positive(message="entrer une valeure positive")
     * @ORM\Column(name="numMaison", type="integer", nullable=false)
     */
    private $nummaison;

    /**
     * @var \User
     *
     * @ORM\ManyToOne(targetEntity="User")
     * @ORM\JoinColumns({
     *   @ORM\JoinColumn(name="iduser", referencedColumnName="idUser")
     * })
     */
    private $iduser;

    public function getIdAdresse(): ?int
    {
        return $this->idAdresse;
    }

    public function getVille(): ?string
    {
        return $this->ville;
    }

    public function setVille(string $ville): self
    {
        $this->ville = $ville;

        return $this;
    }

    public function getRue(): ?string
    {
        return $this->rue;
    }

    public function setRue(string $rue): self
    {
        $this->rue = $rue;

        return $this;
    }

    public function getNummaison(): ?int
    {
        return $this->nummaison;
    }

    public function setNummaison(int $nummaison): self
    {
        $this->nummaison = $nummaison;

        return $this;
    }

    public function getIduser(): ?User
    {
        return $this->iduser;
    }

    public function setIduser(?User $iduser): self
    {
        $this->iduser = $iduser;

        return $this;
    }


}
